### PR TITLE
Claim status page

### PR DESCRIPTION
**Pull request by ynoval-ignw**

### CODE REVIEW

File: client/src/pages/claim-status.tsx [hunk 1]
```diff
@@ -1,63 +1,189 @@
...
+    setSearchText('')
+  }
+  
+  const handleCreate = () => {
+    props.onCreate && props.onCreate()
+  }
+
+  useEffect(() => {
+    props.onSearch && props.onSearch(searchText)
+  }, [searchText])
```
Comment (desmondpnw3dev) [inline client/src/pages/claim-status.tsx:41]: This logic should be handled inside the useClaimStatus hook

File: server/src/services/application.ts [hunk 2]
```diff
@@ -103,23 +104,23 @@ const updateDocumentById = async (collectionName: string, req: Request, res: Res
...
+//         lastModified: fbAdmin.firestore.Timestamp.now()
+//       } as Partial<ApplicationSchema>)
+
+//     return res.status(204).send({
+//       success: true
+//     })
+//   } catch (error) {
+//     res.status(400).json({ error })
+//   }
+// }
```
Comment (desmondpnw3dev) [inline server/src/services/application.ts:123]: Can you delete this if not used?

File: server/src/routes.ts [hunk 3]
```diff
@@ -37,20 +37,23 @@ export const routes = (router: Router) => {
...
+  // router
+  //   .route('/applications/:id')
+  //   .put(decodeToken, isAuthorized, validateApplicationRequest, applicationApi.updateDocumentById.bind(null, ENDPOINTS.NEW_APPLICATIONS))
   router
     .route('/applications/:id')
-    .put(decodeToken, isAuthorized, validateApplicationRequest, applicationApi.updateDocumentById.bind(null, ENDPOINTS.NEW_APPLICATIONS))
-  router
-    .route('/applications/:id')
-    .patch(decodeToken, isAuthorized, validateApplicationRequest, applicationApi.changeDocumentStatusById.bind(null, ENDPOINTS.NEW_APPLICATIONS))
+    // .patch(decodeToken, isAuthorized, validateApplicationRequest, applicationApi.changeDocumentStatusById.bind(null, ENDPOINTS.NEW_APPLICATIONS))
```
Comment (desmondpnw3dev) [inline server/src/routes.ts:48]: What's wrong with the validation? We need this.

File: client/src/pages/claim-status.tsx (hunk 1 above)
Comment (ynoval-ignw) [reply client/src/pages/claim-status.tsx]: Ok, all logic is moved to hook and UI aspect like snackbar is moved from hook to page

**Pull request closed without merging**